salt/metalk8s/salt/minion/configured.sls

```diff
@@ -0,0 +1,19 @@
+{% set saltminion_version = '2018.3.3'%}
```

Comment:
Include this state in the `init.sls` please 

And also we should have a installed file which use this `saltminion_version` to upgrade salt-minion to this specific version